crates/milli/src/search/facet/facet_distribution.rs

```diff
@@ -223,7 +223,7 @@ impl<'a> FacetDistribution<'a> {
                     .index
                     .field_id_docid_facet_strings
                     .get(self.rtxn, &key)?
-                    .unwrap()
+                    .unwrap_or(facet_key)
```

Comment:
Can you add an error log?

Reply:
Fixed, we are now throwing an error log.